gradle.properties

```diff
@@ -20,4 +20,5 @@ kotlin.code.style=official
 # Enables namespacing of each library's R class so that its R class includes only the
 # resources declared in the library itself and none from the library's dependencies,
 # thereby reducing the size of the R class for that library
-android.nonTransitiveRClass=true
\ No newline at end of file
+android.nonTransitiveRClass=true
+android.defaults.buildfeatures.buildconfig=true
```

Comment:
Same question here

Reply:
Upon careful consideration, It appears that this serves the same purpose as `buildConfig = true`.  I should remove it. Thank you.

Reply:
Yep
> I should remove it. Thank you.

👍